package.json

```diff
@@ -9,7 +9,14 @@
     "_metalsmith": "bin/_metalsmith"
   },
   "scripts": {
-    "test": "make test"
+    "pretest": "npm install",
+    "test": "mocha $MOCHA_OPTS",
+    "pretest-debug": "npm install",
+    "test-debug": "mocha debug",
+    "precoverage": "npm install",
+    "coverage": "istanbul cover ./node_modules/.bin/_mocha",
```

Comment:
The Makefile worked by running Istanbul like this: `node --harmony_generators ./node_modules/.bin/istanbul`. Couldn't you just do that?